src/test/java/org/folio/circulation/infrastructure/storage/ServicePointRepositoryTest.java

```diff
@@ -0,0 +1,103 @@
+package org.folio.circulation.infrastructure.storage;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang.StringUtils;
+import org.folio.circulation.domain.*;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.GetManyRecordsClient;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.results.Result;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class ServicePointRepositoryTest {
+
+  @Test
+  void findPrimaryServicePointsForRequestsTest() throws Exception {
+    UUID primaryServicePointId = UUID.randomUUID();
+    String name = "test name";
+
+    Request request = mock(Request.class);
```

Comment:
Similar comment about putting in some blank lines to break up this large block of code.